ToMigrate/Raven.Tests.Issues/RavenDB_505.cs

```diff
@@ -1,29 +0,0 @@
-using Raven.Abstractions.Indexing;
-using Raven.Tests.Common;
-
-using Xunit;
-
-namespace Raven.Tests.Issues
-{
-    public class RavenDB_505 : RavenTest
-    {
-        [Fact]
-        public void CreateDeleteCreateIndex()
```

Comment:
Yes, but this is not a storage engine test. You can remove 'esent' and migrate it without any problems.